### PR TITLE
[DO NOT MERGE] Root cause the np max, min testing flakiness

### PR DESCRIPTION
**Pull request by stu1130**
## Description ##
(Brief description on what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at https://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: tests/python/unittest/test_numpy_op.py [hunk 1]
```diff
@@ -688,6 +688,7 @@ def _test_np_exception(func, shape, dim):
                 for axis in ([i for i in range(in_data_dim)] + [(), None]):
                     for itype in ['float16', 'float32', 'float64', 'int']:
                         # test gluon
+                        print(func, hybridize, keepdims, axis, itype)
```
Comment (reminisce) [inline tests/python/unittest/test_numpy_op.py]: `print(shape)` here?

**Comment by stu1130** [comment]
suspect the problem is in https://github.com/apache/incubator-mxnet/blob/df4125aa1d4ef013e68e6adf3738dabdb1b52865/src/operator/numpy/random/dist_common.h#L156

**Comment by junrushao1994** [comment]
Hey, is this PR relevant to the recent [fix](https://github.com/apache/incubator-mxnet/pull/16415)?

**Comment by stu1130** [comment]
yes, I can close it

**Pull request closed without merging**